generators/server/templates/gradle/_profile_dev.gradle

```diff
@@ -21,20 +21,19 @@ if (project.hasProperty('no-cache')) {
 }
 
 bootRun {
-    args = ["--spring.profiles.active=" + profiles]
+    args = []
 }
 <% if (!skipClient) { %>
 war {
     webAppDirName = '<%= CLIENT_MAIN_SRC_DIR %>'
 }<% } %>
 
-task setProdProperties(dependsOn: bootRun) << {
-    doFirst {
-        System.setProperty('spring.profiles.active', profiles)
-    }
-}
-
 processResources {
+    filesMatching('**/.build-properties.xml') {
```

Comment:
I hate to have an xml for this its kind of ugly :( cant we do the same to the profiles .yml file?  Or using the application.yml file?